### PR TITLE
doc: add minutes for meeting 19 May 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: meetings/2020-05-19.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+## Present
+
+* Michael Dawson (@mhdawson)
+* Lucas Holmquist (@lholmquist)
+* Andrew Hughes (@andrewhughes101)
+* Dominykas
+* Tierney Cyren (@bnb)
+* Owen Buckley (@thescientist13)
+* styfle
+* Andrew Hughes
```
Comment (andrewhughes101) [inline meetings/2020-05-19.md:18]:
```suggestion
```

File: meetings/2020-05-19.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+* Glenn Hinks (@ghinks)
```
Comment (BethGriggs) [inline meetings/2020-05-19.md:19]:
```suggestion
* Glenn Hinks (@ghinks)
* Beth Griggs (@BethGriggs)
```

File: meetings/2020-05-19.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Lucas Holmquist (@lholmquist)
+* Andrew Hughes (@andrewhughes101)
+* Dominykas
+* Tierney Cyren (@bnb)
+* Owen Buckley (@thescientist13)
+* styfle
```
Comment (styfle) [inline meetings/2020-05-19.md:17]:
```suggestion
* Steven (@styfle)
```

File: meetings/2020-05-19.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+* **Recording**: https://www.youtube.com/watch?v=6Es9c9Qk9Qs  
+* **GitHub Issue**: https://github.com/nodejs/package-maintenance/issues/361
+
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Lucas Holmquist (@lholmquist)
+* Andrew Hughes (@andrewhughes101)
+* Dominykas
```
Comment (dominykas) [inline meetings/2020-05-19.md:14]:
```suggestion
* Dominykas (@dominykas)
```

Everyone is doing this!

**Pull request merged**